File: PersonnelManagement/PersonnelManagement/Front/classes/Requests_classes/education_respons.ts
import Person from '../../classes/person';
import Personeducation from '../../classes/personeducation'

export default class Education_respons {
    constructor() {
        this.person = new Person();
        this.education = new Personeducation();
    }
    person: Person;
    education: Personeducation;
}

File: PersonnelManagement/PersonnelManagement/Models/Education_Respons.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PersonnelManagement.Models
{
    public partial class Education_Respons
    {
        public Education_Respons(Person person, Personeducation personeducation)
        {
            Person = person;
            Education = personeducation;
        }
        public Person Person { get; set; }
        public Personeducation Education { get; set; }
    }
}
